src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -129,37 +116,22 @@ export class Dropdown extends Component<DropdownProps> {
     return null;
   }
 
-  dropdownItems = (children: ReactNode, classNames?: { className: string }) =>
-    React.Children.map(
-      children,
-      (child: React.ReactElement<DropdownItemProps>) => {
-        if (React.isValidElement(child)) {
-          return React.cloneElement(child, {
-            ...classNames,
-          });
-        }
-        return child;
-      },
-    );
-
   render() {
     const {
-      id: propId,
+      id,
       name,
       disabled,
       children,
-      labelProps,
       labelText,
-      labelTextProps,
-      labelMode = 'top',
```

Comment:
It seems that **Dropdown** has also the version of having label before the input.
With filterInput we introduced the `labelAlign` prop for that:
https://github.com/vrk-kpa/suomifi-ui-components/blob/3a6e4a804d368f8f2c8f27ba38a480d179d00f8a/src/core/Form/FilterInput/FilterInput.tsx#L63

Reply:
Dropdown previously had only top and hidden as options here so it made sense to switch to visible and hidden in order to make use of the LabelText component. If there's need for more options I think it might make sense to do that separately from this, especially if there are more things to change such as the status text.

Reply:
Vote for separate PR.

Reply:
Created a ticket about this.